fastapi_azure_auth/auth.py

```diff
@@ -328,3 +328,63 @@ def __init__(
             openapi_description=openapi_description,
         )
         self.scheme_name: str = 'Azure AD - PKCE, Multi-tenant'
+
+
+class MultiTenantAzureAuthorizationCodeBearerB2C(AzureAuthorizationCodeBearerBase):
```

Comment:
Is there single tenant B2C tenants? Is there a difference? 

If there isn't, could `B2CAuthorizationCodeBearer` be sufficient?

Reply:
Actually, we haven't tried. We only try with "multi".

Reply:
Did you set it up like [this](https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-create-tenant)? And then just exposed an app reg from your multi tenant tenant into the B2C tenant? 

I'd love a tutorial on your setup to see how you'd do it 😊

Reply:
Put this PR on hold trying to fix tests.

Reply:
@JonasKs Free to continue. I do some refactoring.

Reply:
Yes, it's the same. I will provide the tutorial upcoming week because it's kind of secure information. So I will need some support from our SRE team. :)

Reply:
Awesome! Thank you 😊

Reply:
@JonasKs Sorry for a lot of commits just imports killing me.